docs/functional-php.md

```diff
@@ -107,6 +107,7 @@ if (every($users, function($user, $collectionKey, $collection) {return $user->is
 }
 ```
 
+The default value for $callback is `id()`.
```

Comment:
Can you elaborate on thwat that means? Like "Will return true if every value in the collection is truthy"